include/mbedtls/chacha20.h

```diff
@@ -150,16 +153,16 @@ int mbedtls_chacha20_starts( mbedtls_chacha20_context* ctx,
  *                  key and nonce.
  *
  * \param ctx       The ChaCha20 context to use for encryption or decryption.
+ *                  Must be initialized.
```

Comment:
Suggestion: "It must be initialized and bound to a key and nonce."

(And so on for each function if you decide to adopt that suggestion, I won't repeat it for the other functions and modules.)

Reply:
Ok.